docs/BigQueryTable-batchsource.md

```diff
@@ -39,6 +39,12 @@ Datasets are top-level containers that are used to organize and control access t
 Each record is composed of columns (also called fields).
 Every table is defined by a schema that describes the column names, data types, and other information.
 
+**Partition From Date**: It's inclusive partition start date. It should be a String with format "yyyy-MM-dd". 
+This value is ignored if the table does not support partitioning.
+
+**Partition To Date**: It's inclusive partition end date. It should be a String with format "yyyy-MM-dd". 
```

Comment:
Most of the other plugins that have a start and end time have the start as inclusive and end as exclusive. Let's do the same for this plugin.

Reply:
Done.